app/javascript/app.spec.js

```diff
@@ -0,0 +1,14 @@
+import { shallowMount } from '@vue/test-utils';
+import App from 'app';
+
+describe('App', () => {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'describe' is not defined  no-undef